pkg/webhook/pod/mutating/cluster_colocation_profile.go

```diff
@@ -71,10 +75,11 @@ func (h *PodMutatingHandler) clusterColocationProfileMutatingPod(ctx context.Con
 	}
 
 	for _, profile := range matchedProfiles {
-		err := h.doMutateByColocationProfile(pod, profile)
+		err := h.doMutateByColocationProfile(ctx, pod, profile)
 		if err != nil {
 			return err
```

Comment:
print err

Reply:
Don't need it. Mutating handle will log the error